prow/gerrit/reporter/reporter.go

```diff
@@ -159,36 +158,35 @@ func (c *Client) Report(pj *v1.ProwJob) ([]*v1.ProwJob, error) {
 	selector := labels.Set{
```

Comment:
nit: This is now only used in the `else` block so it would be better to define it there, especially because a selector on an empty report label is a bit misleading if you assume it is used.

Reply:
done